src/main/java/kr/codesqaud/cafe/controller/ArticleController.java

```diff
@@ -16,12 +20,30 @@ public ArticleController(ArticleService articleService) {
         this.articleService = articleService;
     }
 
+    @GetMapping("/qna/article")
+    public String askQuestion(HttpSession httpSession) {
```

Comment:
```suggestion
    @GetMapping("/qna/articleForm")
    public String askQuestionForm(HttpSession httpSession) {
```

form 을 리턴해 주는 경우 명시적으로 URL 등에  form 이 들어가는 게 좋을 것 같아요.